content/docs/csidriver/release/unity.md

```diff
@@ -36,7 +36,7 @@ description: Release notes for Unity XT CSI driver
 | A CSI ephemeral pod may not get created in OpenShift 4.13 and fail with the error `"error when creating pod: the pod uses an inline volume provided by CSIDriver csi-unity.dellemc.com, and the namespace has a pod security enforcement level that is lower than privileged."` | This issue occurs because OpenShift 4.13 introduced the CSI Volume Admission plugin to restrict the use of a CSI driver capable of provisioning CSI ephemeral volumes during pod admission. Therefore, an additional label `security.openshift.io/csi-ephemeral-volume-profile` in [csidriver.yaml](https://github.com/dell/helm-charts/blob/csi-unity-2.8.0/charts/csi-unity/templates/csidriver.yaml) file with the required security profile value should be provided. Follow [OpenShift 4.13 documentation for CSI Ephemeral Volumes](https://docs.openshift.com/container-platform/4.13/storage/container_storage_interface/ephemeral-storage-csi-inline.html) for more information. |
 | If the volume limit is exhausted and there are pending pods and PVCs due to `exceed max volume count`, the pending PVCs will be bound to PVs and the pending pods will be scheduled to nodes when the driver pods are restarted. | It is advised not to have any pending pods or PVCs once the volume limit per node is exhausted on a CSI Driver. There is an open issue reported with Kubernetes at https://github.com/kubernetes/kubernetes/issues/95911 with the same behavior. |
 | fsGroupPolicy may not work as expected without root privileges for NFS only [https://github.com/kubernetes/examples/issues/260](https://github.com/kubernetes/examples/issues/260) | To get the desired behavior set “RootClientEnabled” = “true” in the storage class parameter |
-| Controller publish is taking too long to complete/ Health monitoring is causing Unity array to panic by opening multiple sessions | Disable Volume health monitoring on the node and keep it only at the controller level.|
+| Controller publish is taking too long to complete/ Health monitoring is causing Unity array to panic by opening multiple sessions | Disable Volume health monitoring on the node and keep it only at the controller level. Refer [here](https://dell.github.io/csm-docs/docs/csidriver/features/unity/#volume-health-monitoring) for more information about enabling/disabling volume health monitoring|
```

Comment:
"... As a workaround, Disable Volume health... " - Disable should be disable and Volume should be volume. What other information can be provided besides "Controller publish is taking too long?". What about things that would be observed in the logs? Under what circumstances would this be seen?

Reply:
@gallacher :  Health monitoring is causing Unity array to panic by opening multiple sessions - This is the reason why it has been seen, it is also added

Reply:
@shanmydell - what errors are seen in the logs? What's observed in k8s? Please provide more details.

Reply:
Thanks @shanmydell for mentioning the error we are receiving in k8s logs, reason and the roundabout to circumvent the issue.